lib/commands/clipboard.js

```diff
@@ -0,0 +1,40 @@
+import { exec } from 'teen_process';
+
+const commands = {};
+
+/**
+ * Sets the clipboard's content on the Windows PC.
+ *
+ * @param {!string} content - The content to be set as base64 encoded string.
+ * @param {?string} contentType [plaintext] - The type of the content to set.
+ *                                            Only `plaintext` and 'image' are supported.
+ */
+commands.setClipboard = async function setClipboard (content, contentType) {
+  if (contentType === 'plaintext') {
```

Comment:
I'd pipe the `content` to powershell

Reply:
Otherwise the exec command will be broken if your content string contains single quotes

Reply:
or if the string is too long

Reply:
See https://2ality.com/2018/05/child-process-streams.html for example. `exec` is just a wrapper over `spawn` call:  https://github.com/appium/node-teen_process/blob/master/lib/exec.js

Reply:
The other less advanced approach is to write the content to a temporary file and feed the file to PS, then delete it

Reply:
I would rather go for the piping, but quite novice, and not sure how to do this given the commands for PS expect variables and don't seem to be friendly to piping... Any advice?

Reply:
how about something like this: https://stackoverflow.com/questions/38667584/getting-stdin-into-the-powershell-stream